packages/ckeditor5-table/lang/contexts.json

```
@@ -52,9 +52,11 @@
 	"Align cell text to the top": "The label used by assistive technologies describing a button that aligns the table cell text to the top.",
 	"Align cell text to the middle": "The label used by assistive technologies describing a button that aligns the table cell text to the middle.",
 	"Align cell text to the bottom": "The label used by assistive technologies describing a button that aligns the table cell text to the bottom.",
-	"Align table to the left": "The label used by assistive technologies describing a button that aligns the table to the left.",
-	"Center table": "The label used by assistive technologies describing a button that centers the table.",
-	"Align table to the right": "The label used by assistive technologies describing a button that aligns the table to the right.",
+	"Align table to the left (inline, allows text wrapping)": "The label used by assistive technologies describing a button that aligns the table to the left as an inline element, allowing text to wrap around it.",
+	"Center table (block, no text wrapping)": "The label used by assistive technologies describing a button that centers the table as a block element with no text wrapping.",
+	"Align table to the right (inline, allows text wrapping)": "The label used by assistive technologies describing a button that aligns the table to the right as an inline element, allowing text to wrap around it.",
+	"Align table to the left (block, no text wrapping)": "The label used by assistive technologies describing a button that aligns the table to the left as a block element with no text wrapping.",
+	"Align table to the right (block, no text wrapping)": "The label used by assistive technologies describing a button that aligns the table to the right as a block element with no text wrapping.",
```

I think that way is more natural.

```suggestion
	"Align table to the left with text wrapping": "The label used by assistive technologies describing a button that aligns the table to the left as an inline element, allowing text to wrap around it.",
	"Center table with no text wrapping": "The label used by assistive technologies describing a button that centers the table as a block element with no text wrapping.",
	"Align table to the right with text wrapping": "The label used by assistive technologies describing a button that aligns the table to the right as an inline element, allowing text to wrap around it.",
	"Align table to the left with no text wrapping": "The label used by assistive technologies describing a button that aligns the table to the left as a block element with no text wrapping.",
	"Align table to the right with no text wrapping": "The label used by assistive technologies describing a button that aligns the table to the right as a block element with no text wrapping.",
```